src/app/workflow/register-workflow-modal/register-workflow-modal.component.html

```diff
@@ -100,13 +100,13 @@ <h4 class="modal-title">Register Workflow</h4>
                   <div class="input-group">
                     <div class="input-group-btn">
                       <button type="button" class="btn btn-default btn-sm dropdown-toggle" data-toggle="dropdown">
-                        {{workflow?.descriptorType}}
+                        {{workflow?.descriptorType.toUpperCase()}}
                         <span class="caret"></span>
                       </button>
                       <ul class="dropdown-menu">
                         <li *ngFor="let descriptorType of (descriptorLanguages$ | async)">
                           <a class="dropdown-item" (click)="changeDescriptorType(descriptorType.toLowerCase())">
-                            {{descriptorType | lowercase}}
+                            {{descriptorType | uppercase}}
```

Comment:
Same comment as below, regarding NEXTFLOW being in caps.

Reply:
Same reason